src/sentry/api/endpoints/organization_dynamic_sampling_sdk_versions.py

```diff
@@ -37,9 +38,36 @@
 )
 
 
+class QueryBoundsException(Exception):
+    pass
+
+
 class OrganizationDynamicSamplingSDKVersionsEndpoint(OrganizationEndpoint):
     private = True
 
+    @staticmethod
+    def __parse_query_bounds(query_start, query_end):
+        if not query_start or not query_end:
+            raise QueryBoundsException("start and end are required")
+
+        query_start = ensure_aware(parse_date(query_start))
+        query_end = ensure_aware(parse_date(query_end))
+
+        if query_start > query_end:
+            raise QueryBoundsException("start has to be before end")
+
+        if query_end - query_start > timedelta(days=1):
+            raise QueryBoundsException("start and end have to be a maximum of 1 day apart")
```

Comment:
```suggestion
            raise QueryBoundsException("`start` and `end` have to be a maximum of 1 day apart")
```

Reply:
Why?

Reply:
it's common practise to decouple from string columns/fields. But if we have in sentry another code convention i'm okey to stay with it.
Example from DRF docs:
```
{"status_code": 405, "detail": "Method 'DELETE' not allowed."}
```
Just visually to show that DELETE is not verb delete but it's HTTP verb.

Reply:
I will add single quotes, backticks seem strange to me to be used here